stable/ibm-istio/charts/tracing/templates/ingress-jaeger.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
     {{- end }}
 spec:
   rules:
-{{- if .Values.ingress.hosts }}
+{{- if .Values.tracing.jaeger.ingress.hosts }}
     {{- range $host := .Values.jaeger.ingress.hosts }}
```

Comment:
You should also change this to `.Values.tracing.jaeger.ingress.hosts`

Reply:
@aliartiza75

Reply:
@waseem-h comment has been addressed